### PR TITLE
Add patch endpoint for payment instruments

### PR DESCRIPTION
**Pull request by martyna-lachut-cko**
# Description of changes in PR

[//]: # 'Please add your description here'
Added documentation for updating payment instruments (PATCH endpoint)

## Checklist

- [X] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

File: nas_spec/changelog.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 | Date       | Description of change                                                                                                 
 |------------|-----------------------------------------------------------------------------------------------------------------------|
-| 2022/12/15 | Change Platforms schedule `by_day` response samples to use arrays                                                       |
+| 2022/12/20 | Add Platforms Update Payment Instrument Request spec                                                                            |
```
Comment (gareth-forshaw-cko) [inline nas_spec/changelog.md:5]: Minor but "." would be more consistent.

File: nas_spec/paths/accounts@entities@{entityId}@payment-instruments@{id}.yaml [hunk 2]
```diff
@@ -59,3 +59,39 @@ get:
...
+        schema:
+          $ref: '#/components/schemas/PlatformsPaymentInstrumentUpdate'
+  security:
+    - OAuth:
+        - accounts
+    - ApiSecretKey:
+        - accounts
+  responses:
+    '200':
+      description: Payment Instrument updated successfully
```
Comment (gareth-forshaw-cko) [inline nas_spec/paths/accounts@entities@{entityId}@payment-instruments@{id}.yaml:78]: Not sure if this is hardcoded or not, but if not, ideally lower case "i", "Payment **i**nstrument"

**Review by gareth-forshaw-cko — CHANGES_REQUESTED** [review]
Minor comment but looks good.

**Review by gareth-forshaw-cko — DISMISSED** [review]

File: nas_spec/changelog.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 | Date       | Description of change                                                                                                 
 |------------|-----------------------------------------------------------------------------------------------------------------------|
+| 2022/12/23 | Add Platforms Update Payment Instrument Request spec.                                                                 |
```
Comment (natasha-moore-cko) [inline nas_spec/changelog.md:5]:
```suggestion
| 2023/01/03 | Add Platforms Update Payment Instrument Request spec.                                                                 |
```

**Review by natasha-moore-cko — CHANGES_REQUESTED** [review]

**Review by natasha-moore-cko — APPROVED** [review]

**Pull request merged**